server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -1544,7 +1546,7 @@ private SetMonitorServiceCommand createMonitorServiceCommand(DomainRouterVO rout
         final SetMonitorServiceCommand command = new SetMonitorServiceCommand(services);
         command.setAccessDetail(NetworkElementCommand.ROUTER_IP, getRouterControlIP(router));
         command.setAccessDetail(NetworkElementCommand.ROUTER_NAME, router.getInstanceName());
-        command.setAccessDetail(SetMonitorServiceCommand.ROUTER_HEALTH_CHECKS_ENABLED, RouterHealthChecksEnabled.valueIn(router.getDataCenterId()).toString());
+        command.setAccessDetail(SetMonitorServiceCommand.ROUTER_HEALTH_CHECKS_ENABLED, RouterHealthChecksEnabled.value().toString());
```

Comment:
why remove the zone parameter, now that we moved the scope to zone?

Reply:
same as above. The previous implementation was buggy.